InvenTree/templates/js/translated/part.js

```diff
@@ -329,15 +329,32 @@ function deletePartCategory(pk, options={}) {
     var html = `
     <div class='alert alert-block alert-danger'>
     {% trans "Are you sure you want to delete this part category?" %}
-    <ul>
-        <li>{% trans "Any child categories will be moved to the parent of this category" %}</li>
-        <li>{% trans "Any parts in this category will be moved to the parent of this category" %}</li>
-    </ul>
     </div>`;
-
+    var subChoices = [
+        {
+            value: 'move_to_parent',
```

Comment:
I'm not a huge fan of passing string variables through the API and then comparing them, especially when these are essentially "boolean" checks.

I think a *cleaner* approach would be have boolean inputs for:

- delete parts
- delete child categories

And then pass true or false for these named parameters. Makes the backend cleaner too.

Reply:
> I'm not a huge fan of passing string variables through the API and then comparing them, especially when these are essentially "boolean" checks.

I second the usage of booleans, however my impression is that the choice is the most intuitive way to present these decisions to the user. 

What shall we write to the label of the checkbox? 

If we write just "Delete parts" then effect of the turned off value will not be defined for the user. 
Writing "Delete parts (if turned off the parts will be moved to parent)" to the labels looks a bit weird for me.

tldr: I would keep the choices, but with True and False values as options.

Reply:
I don't think that these are booleans. Because if delete child's is true, move to parent can't be also true. So they are dependent. What if we later came up with a third deletion strategy? What about changing it from strings to numbers (move to parent -> 1, recursive delete ->2, ...)? Or is that an even dirtier solution than the strings?

Reply:
@maksimstojkovic I think your approach is a reasonable compromise. 

Regarding the "incompatible choices" raised by @luwol03 - the back-end should check that the choices are compatible and if not, raise an error rather than actually deleting anything. This will render an error on the form so the user gets feedback